kube-fluentd/gcp/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For systems without journald
+mkdir -p /var/log/journal
+
+# If you attach host lib directory, this will copy the systemd lib
+# This may be necessary in order for journald to work
+if [ ! -z "`ls /host/lib/libsystemd* 2>/dev/null`" ]
+then
+  rm /lib/x86_64-linux-gnu/libsystemd*
+  cp -a /host/lib/libsystemd* /lib/x86_64-linux-gnu/
+fi
+
+# Jemalloc is a better malloc implementation
+LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.1
+
+fluentd $@
```

Comment:
The problem with that, as with `--no-supervisor` is that someone might want to override it, I think it's better to be flexible here. Also, this should not be the problem, because the dependencies are fixed, implicit conventions should not change across patch releases